src/core/services/LabService.ts

```diff
@@ -229,7 +229,7 @@ export class LabServiceImpl implements LabService {
           pricePerShare: pJarRatio.toString(),
           apy: { ...pJarData.apy, net_apy: toBN(performance.toString()).dividedBy(100).toNumber() },
           displayName: 'pSLPyvBOOST-ETH',
-          displayIcon: `https://raw.githubusercontent.com/yearn/yearn-assets/master/icons/tokens/${PSLPYVBOOSTETH}/logo-128.png`,
+          displayIcon: `https://raw.githubusercontent.com/yearn/yearn-assets/master/icons/multichain-tokens/1/${PSLPYVBOOSTETH}/logo-128.png`,
```

Comment:
should we save this string as a const?

Reply:
https://raw.githubusercontent.com/yearn/yearn-assets/master/icons/multichain-tokens/1/